config/config_phpstan.yml

```diff
@@ -0,0 +1,43 @@
+imports:
+    - { resource: security.yml }
```

Comment:
Do you think it would be possible to use the `security.yml` file from the core-bundle? Then we would not have to manually sync it.

Reply:
That should be possible. I will have a look at it.

Reply:
The core-bundle has no `security.yml` file, but the `manager-bundle` has it in the skeleton. Should we take the one from there?

Reply:
Yes.

Reply:
Fixed in 541851a.